### PR TITLE
For #921: 0crat ignores debt on new payment.

### PR DESCRIPTION
**Pull request by paulodamaso**
For #921: 0crat ignores debt on new payment:
- Added bundle test to assure that 0crat add new payment to debts if user already has a debt, as defined in https://github.com/zerocracy/farm/issues/921#issuecomment-414080640.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1626](https://github.com/zerocracy/farm/issues/1626) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1626](https://github.com/zerocracy/farm/issues/1626) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2330758728); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/resources/com/zerocracy/bundles/add_payment_to_debts_if_has_debt/_before.groovy [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import com.zerocracy.Farm
+import com.zerocracy.Project
+import com.zerocracy.cash.Cash
+import com.zerocracy.entry.ExtGithub
+import com.zerocracy.pm.cost.Rates
+import com.zerocracy.pmo.Debts
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
+  Github github = new ExtGithub(farm).value()
```
Comment (krzyk) [inline src/test/resources/com/zerocracy/bundles/add_payment_to_debts_if_has_debt/_before.groovy]: @paulodamaso you can inline `github`

**Comment by paulodamaso** [comment]
@krzyk Done, please take a look.

**Comment by paulodamaso** [comment]
@krzyk Thanks, corrected, please take a look,

**Comment by krzyk** [comment]
@paulodamaso thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@g4s8 ping

File: src/test/resources/com/zerocracy/bundles/add_payment_to_debts_if_has_debt/_before.groovy [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import com.zerocracy.Project
+import com.zerocracy.cash.Cash
+import com.zerocracy.entry.ExtGithub
+import com.zerocracy.pm.cost.Rates
+import com.zerocracy.pmo.Debts
+
+def exec(Project project, XML xml) {
+  Farm farm = binding.variables.farm
+  new ExtGithub(farm).value().repos().create(new Repos.RepoCreate('test', false))
+  new Rates(project).bootstrap().set('paulodamaso', new Cash.S('$100'))
```
Comment (g4s8) [inline src/test/resources/com/zerocracy/bundles/add_payment_to_debts_if_has_debt/_before.groovy]: @paulodamaso why this code is needed? If you submit `make payment` claim with `cash` it just send this amount to user ignoring `rates.xml`, or I miss something?

File: src/test/resources/com/zerocracy/bundles/add_payment_to_debts_if_has_debt/_before.groovy [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  new Debts(farm).bootstrap().add('paulodamaso', new Cash.S('$5'), 'Old debt', 'Any reason')
```
Comment (g4s8) [inline src/test/resources/com/zerocracy/bundles/add_payment_to_debts_if_has_debt/_before.groovy]: @paulodamaso same here, you can define it in `debts.xml`

**Comment by paulodamaso** [comment]
@g4s8 Done, please take a look.

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1626](https://github.com/zerocracy/farm/issues/1626) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**